### ISSUE TITLE
Create an icon for Content Warning

### ISSUE THREAD

**Issue by xuv** [ForkAwesome/Fork-Awesome#27]
I really not have much idea of how it should look like.
But I see an important and widely used case in Mastodon software.
They have an issue about it: https://github.com/tootsuite/mastodon/issues/1249

**Comment by micahilbery** [comment]
Maybe something like an image icon with an exclamation point emblem on it. 🖼️⚠️

**Comment by xuv** [comment]
Yes. It's a start. Though, it's not about images. It's about text. 

And the conversation they are having in the Mastodon issue is more about signifying "hiding the text", and opposite would be to "reveal it", that they want to convey. If I understand it well.

**Comment by xuv** [comment]
Ok. I'm switching back and forth the topic of this issue, which is not a good idea. But here is a more precise request with an example.
https://github.com/FortAwesome/Font-Awesome/issues/10911

And I think if we could achieve something closer to the actual [fa-flag](https://forkawesome.github.io/Fork-Awesome/icon/flag/) look, that could work.

**Comment by Sylvhem** [comment]
I'm not to sure about that… To me it looks more like a symbol to indicate the emplacement of something.

**Comment by xuv** [comment]
@Sylvhem You are totally right. I was mostly following @nclm request in [this issue (comment)](https://github.com/FortAwesome/Font-Awesome/issues/10911#issuecomment-292710800)

Do you have suggestions? Or ideas we could explore?

**Comment by Sylvhem** [comment]
@xuv As I am not a designer myself, I'm not sure to be qualified to work on this kind of things. But since we want to display a warning, I think it would be good if the symbol for content warning was related to the warning symbol: ⚠.

**Comment by xuv** [comment]
@Sylvhem There is already these types of icons in the Fork Awesome set. 
There is the triangle one: https://forkawesome.github.io/Fork-Awesome/icon/exclamation-triangle/

But also, maybe nice, circle warning one:
![screenshot-2018-2-25 fa-exclamation-circle fork awesome icons](https://user-images.githubusercontent.com/192539/36653923-ef1ae01c-1a86-11e8-875c-688349f0c49a.png)
https://forkawesome.github.io/Fork-Awesome/icon/exclamation-circle/

**Comment by jeroenpraat** [comment]
What about a circle with a ellipses below an explanation mark? Something like a the circle above, but with …

**Comment by jeroenpraat** [comment]
Mock-up:

![schermafdruk van 2019-02-21 22-35-49](https://user-images.githubusercontent.com/41594439/53203831-79b5e180-362a-11e9-97a9-e05de87c5d5b.png)

**Comment by xuv** [comment]
@jeroenpraat Interesting. Thanks for sharing. I'm not sure I understand the meaning of this. And what is the difference in meaning from the plain '!' in a circle.

**Comment by jeroenpraat** [comment]
The … is the hidden text and the ! is the warning.

**Comment by gabiturra** [comment]
Is this an idea? An once you open the text the eye wouldn't have the exclamation point. so it can go back and forth...

![eye-exclamation](https://user-images.githubusercontent.com/64840610/118978676-dd4d2780-b977-11eb-8f38-7a1319098bfb.png)
